src/EnergyPlus/PhotovoltaicThermalCollectors.hh

```diff
@@ -198,6 +273,12 @@ namespace PhotovoltaicThermalCollectors {
 
     int GetAirOutletNodeNum(EnergyPlusData &state, std::string_view PVTName, bool &ErrorsFound);
 
+    void GetPVTmodelIndex(EnergyPlusData &state, int const SurfacePtr, int &PVTIndex);
+
+    void SetPVTQdotSource(EnergyPlusData &state, int const PVTNum, Real64 const QSource);
+
+    void GetPVTTsColl(EnergyPlusData &state, int const PVTNum, Real64 &TsColl);
```

Comment:
Return `TsCoil` through the return value.

Reply:
we kept this as is. We followed the same approach as for 'CellIntegration::TranspiredCollector'.